src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/What-Language.cshtml.cs

```diff
@@ -177,12 +178,15 @@ protected override IActionResult OnPostWithModel()
             }
             else if (button.StartsWith("remove"))
             {
-                int indexToRemove = int.Parse(button.Substring("remove-".Length));
-                languageCodes = languageCodes.Where((_, i) => i != indexToRemove);
-
-                if (!languageCodes.Any())
+                var indexToRemove = int.Parse(button.Substring("remove-".Length));
```

Comment:
Possibly with this chunk too, it's hard to tell what it's doing at face value without a method name to describe it

Reply:
I've moved the code out into a seperate method now